config/config-passport.js

```diff
@@ -0,0 +1,26 @@
+const passport = require('passport');
+const passportJWT = require('passport-jwt');
+const User = require('../models/user');
+require('dotenv').config();
+const secret = process.env.SECRET;
+
+const ExtractJWT = passportJWT.ExtractJwt;
+const Strategy = passportJWT.Strategy;
+const params = {
+  secretOrKey: secret,
+  jwtFromRequest: ExtractJWT.fromAuthHeaderAsBearerToken(),
+};
+
+// JWT Strategy
```

Comment:
no shit, Sherlock! - jak by nie używaj komentarzy jeśli nie musisz, a jeśli musisz to znaczy że coś zwaliłeś w kodzie i że trzeba to wyjaśniać (a wtedy zamiast wyjaśniać powinneś jeszcze raz napisać ten kod by nie było trzeba). Jedyne komentarze które zostawiam to jak używam jakiegoś dziwnego obejścia w fremoworku/libce. To wtedy w komentarzu zostawiam link do github issues / repo gdzie znalazłem obejście aby ktoś mógł się dowiedzieć czemu tak i po co to.